### PR TITLE
Usecase

### PR DESCRIPTION
**Pull request by srayagarwal**
Signed-off-by: Sray Agarwal <sagarwal54@sapient.com>

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,8 +3,18 @@ matplotlib>=3.2.1
...
 pandas>=0.25.1
 scikit-learn>=0.22.1
+
+
+math
+datetime
+operator
+io
+seaborn
+scipy.stats
```
Comment (romanlutz) [inline requirements.txt:13]: Isn't this part of scipy automatically?

File: requirements.txt [hunk 2]
```diff
@@ -37,5 +47,16 @@ shap
...
 sphinx
+
+
+
+
+
+
+
 sphinx-gallery
+<<<<<<< HEAD
```
Comment (romanlutz) [inline requirements.txt:58]: Some sort of merge conflict that wasn't resolved

File: test/notebooks/functions.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+def stat_parity_test(data, protected_attribute_name, label, fav, privileged_group_protected_attribute_value, unprivileged_group_protected_attribute_value):
+    priv_subset = data[data[protected_attribute_name] == privileged_group_protected_attribute_value]
+    priv_count_pos = len(priv_subset[priv_subset[label] == fav])
+    unpriv_subset = data[data[protected_attribute_name] == unprivileged_group_protected_attribute_value]
+    unpriv_count_pos = len(unpriv_subset[unpriv_subset[label] == fav])
+    total_priv = len(priv_subset)
+    total_unpriv = len(unpriv_subset)
+    stat_parity = (unpriv_count_pos / total_unpriv) - (priv_count_pos / total_priv)
+    disprate_impact = (unpriv_count_pos / total_unpriv) / (priv_count_pos / total_priv)
```
Comment (romanlutz) [inline test/notebooks/functions.py:89]: nit: typo "disparate" as opposed to "disprate"

**Comment by romanlutz** [comment]
Where is this data from by the way? Usually we try to set things up so that there just 1 file (the notebook) and you don't need to download lots of others to make it work. I have a separate package (`tempeh`) which provides the code to download datasets from various places. We can add it there if you want, but it also needs to be clear that we didn't copy it from anywhere. That's why I usually prefer just writing download code from somebody else's location.

**Comment by srayagarwal** [comment]
I downloaded the data from an open source platform and then edited it to fit the purpose. The data now being used is in the notebook folder.

**Comment by srayagarwal** [comment]
We are not using functions.py anymore.
The new pull request has no functions.py or fairlearn.py

**Pull request closed without merging**